src/olympia/devhub/feeds.py

```diff
@@ -31,7 +30,7 @@ def items(self, key):
         if key.addon:
             addons = key.addon
         else:  # We are showing all the add-ons
-            addons = Addon.objects.filter(authors=key.user)
+            addons = key.user.addons.all()
```

Comment:
Is there a test for this change? Is it covered by the change in ` src/olympia/devhub/tests/test_views.py`?

Reply:
yeah, I added to the `setUp` for the base `HubTest` which is used by the feeds tests too.